packages/manager/apps/web/client/app/hosting/hosting.bundle.js

```diff
@@ -23,7 +23,7 @@ import './cron/delete/hosting-cron-delete.controller';
 import './cron/hosting-cron.constants';
 import './cron/hosting-cron.controller';
 import './cron/hosting-cron.service';
-import './database/add/add.constants';
+import './database/add/hosting-database-add.constants';
```

Comment:
Are `EXTRA_SQL_PERSO` & `MAX_USER_LENGTH` used elsewhere ? If not, we could remove this import.

More theses bundle files are empty, more we're happy 😄

Reply:
done !